src/core_plugins/elasticsearch/lib/migrate_config.js

```diff
@@ -3,7 +3,8 @@ import { mappings } from './kibana_index_mappings';
 
 module.exports = function (server) {
   const config = server.config();
-  const client = server.plugins.elasticsearch.client;
+  const callAsKibanaUser = server.plugins.elasticsearch.getCluster('admin').callAsKibanaUser;
```

Comment:
destructuring the `callAsKibanaUser` property here, and other places, could trim down the line length a bit. Something I would personally appreciate.